tests/house_tests.py

```diff
@@ -9,99 +9,107 @@
-        HouseFactory(
-            country=Country.objects.get(name='Israel'),
-            city=City.objects.get(name='Tel Aviv'),
-            income=5000,
-            children=1,
-            parent_profession_1=Job.OTHER,
-        ).save()
-        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
-        assert len(houses) == 1
-
-    def test_filter_country(self):
-        pass_filter_house_create()
-        HouseFactory(
-            country=Country.objects.get(name='India'),
-            city=City.objects.get(name='Tel Aviv'),
-            income=5000,
-            children=1,
-            parent_profession_1=Job.TEACHER,
-        ).save()
-        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
-        assert len(houses) == 1
-
-    def test_filter_city(self):
-        pass_filter_house_create()
-        HouseFactory(
-            country=Country.objects.get(name='Israel'),
-            city=City.objects.get(name='Ramat Gan'),
-            income=5000,
-            children=1,
-            parent_profession_1=Job.TEACHER,
-        ).save()
-        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
-        assert len(houses) == 1
-
-    def test_filter_highest_income(self):
-        pass_filter_house_create()
-        HouseFactory(
-            country=Country.objects.get(name='Israel'),
-            city=City.objects.get(name='Tel Aviv'),
-            income=12000,
-            children=1,
-            parent_profession_1=Job.TEACHER,
-        ).save()
-        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
-        assert len(houses) == 1
-
-    def test_filter_lowest_income(self):
-        pass_filter_house_create()
-        HouseFactory(
-            country=Country.objects.get(name='Israel'),
-            city=City.objects.get(name='Tel Aviv'),
-            income=4000,
-            children=1,
-            parent_profession_1=Job.TEACHER,
-        ).save()
-        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
-        assert len(houses) == 1
-
-    def test_profession_either_parent(self):
-        HouseFactory(
-            country=Country.objects.get(name='Israel'),
-            city=City.objects.get(name='Tel Aviv'),
-            income=5000,
-            children=1,
-            parent_profession_2=Job.TEACHER,
-        ).save()
-        HouseFactory(
-            country=Country.objects.get(name='Israel'),
-            city=City.objects.get(name='Tel Aviv'),
-            income=5000,
-            children=1,
-            parent_profession_1=Job.TEACHER,
-        ).save()
-        houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
-        assert len(houses) == 2
-        assert House.objects.get(parent_profession_1=Job.TEACHER) in houses
-        assert House.objects.get(parent_profession_2=Job.TEACHER) in houses
+def test_save_house(db):
```

Comment:
Shouldn't these test be included in a proper Test Class?
Like we did :
@pytest.mark.django_db
class TestFilterHouseForm:
   def test1
   def test 2
   ...

Reply:
So no, there is no need and actually its better to use Pytest functionally. some classes that use the country and cities didnt work after I changed the way we load the initial data, so they had to migrate to be functional tests to use the loaddata fixture that makes sure it loads the initial data before the tests